src/plone/api/tests/test_content.py

```diff
@@ -732,6 +732,10 @@ def test_delete_multiple(self):
         self.assertNotIn('copy_of_about', container)
         self.assertNotIn('about', container['events'])
 
+    def test_delete_no_objs(self):
+        # Check that we allow passing in an empty list of objects.
+        api.content.delete(obj=None, objects=[])
```

Comment:
this test is wrong as `obj` and `objects` are mutually exclusive parameters:

    @mutually_exclusive_parameters('obj', 'objects')

Reply:
The test passes, so apparently it is fine.
I see that the `mutually_exclusive_parameters` decorator uses [`_get_supplied_args`](https://github.com/plone/plone.api/blob/master/src/plone/api/validation.py#L30), which ignores arguments that are `None`. That seems why it is okay.